service-front/module/Application/src/Controller/Authenticated/Lpa/StatusController.php

```diff
@@ -45,9 +45,9 @@ public function indexAction()
         // Return either the applicationRejectedDate or the applicationRegistrationDate based on what's received from Sirius
         $metadata = $lpa->getMetadata();
 
-        if ($metadata['application-rejected-date']!= null)
+        if (array_key_exists('application-rejected-date', $metadata) && $metadata['application-rejected-date'] != null)
```

Comment:
Array lookup method which can deal with keys not existing.

Reply:
isset() behaves in the way we require: returns false if the key is not set *or* if it is set but is null.